loqusdbapi/main.py

```diff
@@ -139,3 +143,26 @@ def read_case(case_id: str, db: MongoAdapter = Depends(database)):
             status_code=status.HTTP_404_NOT_FOUND, detail=f"Case {case_id} not found"
         )
     return case
+
+
+@app.delete("/cases/{case_id}")
+def delete_case(case_id: str, db: MongoAdapter = Depends(database)):
+    existing_case = db.case({"case_id": case_id})
+    if not existing_case:
+        return JSONResponse(f"Case {case_id} does not exist", status_code=404)
```

Comment:
```suggestion
        return JSONResponse(f"Case {case_id} does not exist", status_code=NOT_FOUND)
```
Would be to make the status codes into named constants - more readable and you do not have to remember what they mean